samples/snippets/src/test/java/aiplatform/ExportModelSampleTest.java

```diff
@@ -35,7 +35,7 @@
 public class ExportModelSampleTest {
 
   private static final String PROJECT_ID = "ucaip-sample-tests";
-  private static final String MODEL_ID = "5359002081594179584";
+  private static final String MODEL_ID = "3422489426196955136";
```

Comment:
can we reuse or create a new env var for this?

Reply:
Acknowledged.